Packs/Active_Directory_Query/ReleaseNotes/1_6_24.md

```diff
@@ -0,0 +1,14 @@
+
+#### Integrations
+
+##### Active Directory Query v2
+
+- Added the ***ad-test-credentials*** command.
+- Updated the Docker image to: *demisto/ldap:2.9.1.75217*.
+
+#### Scripts
+
+##### New: ADGetUser
+
+- New: Deprecated. Use the ad-get-user command in the Active Directory v2 integration instead.account['Groups'] = demisto.get( Use Active Directory to retrieve detailed information about a user account. The user can be specified by name, email or as an Active Directory Distinguished Name (DN).
+If no filter is provided, the result will show all users. (Available from Cortex XSOAR 6.8.0).
```

Comment:
I believe this isn't actually a new integration, but just a split of the existing unified one?
If so, please fix that notes here, and just write what you've added (if there's anything you did besides just splitting it) since the structure of the files is invisible and irrelevant to end-users.

Reply:
I didn't make any changes to ADGetUser.